src/tilemap/TileLayer.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System.Threading.Tasks.Dataflow;
+
+namespace Tile_Mapper.tilemap
+{
+    //Class to store and manipulate information related to tile layers
+    public class TileLayer
+    {
+        //Variables
+
+        private string tileSet;
+        private int[,] tilePlacements;
+
+        //Constructor for new layer
+        public TileLayer(uint rows, uint columns, string tileSet)
+        {
+            this.tileSet = tileSet;
+            this.tilePlacements = new int[rows,columns];
+
+            //Initialize a blank layer
+            for (int i = 0; i < rows; i++)
+            {
+                for (int j = 0; j < columns; j++)
+                {
+                    this.tilePlacements[i,j] = -1;
+                }
+            }
+        }
+
+        //Constructor for loaded layer
+        public TileLayer(int[,] tiles, string tileSet)
+        {
+            this.tileSet = tileSet;
+            this.tilePlacements = tiles;
+        }
+
+        //Method to get tile at location
+        public int GetTile(uint x, uint y)
+        {
+            return this.tilePlacements[x,y];
```

Comment:
You don't have to start accessing variables with `this.` when it is not needed. Optional to change this.